src/ui/component/sideBar/view.jsx

```diff
@@ -1,4 +1,6 @@
 // @flow
```

Comment:
This is the other big file. No longer bringing in the nav links from redux. Will add the tutorial back in a future PR.